cognite/client/utils/_identifier.py

```diff
@@ -158,10 +157,20 @@ def as_primitives(self) -> list[int | str]:
     def are_unique(self) -> bool:
         return len(self) == len(set(self.as_primitives()))
 
+    @staticmethod
+    def unwrap_identifier(identifier: Union[str, int, Dict]) -> Union[str, int]:
```

Comment:
Maybe save for later but
```suggestion
    def unwrap_identifier(identifier: str | int | dict) -> Union[str, int]:
```